src/pydantic_mermaid/models.py

```diff
@@ -53,6 +54,8 @@ def generate_class(self, exclude: Set[str]) -> str:
         # flake8 of python3.12 treats class as a keyword and gives the following error:
         # ./src/pydantic_mermaid/models.py:40:17: E272 multiple spaces before keyword
         s = f"\n{'    '}class {self.name} {{\n"
+        if self.annotation:
```

Comment:
As defined in the [mermaid docs](https://mermaid.js.org/syntax/classDiagram.html#annotations-on-classes)